hub/auto/tests/test_ingestion.py

```diff
@@ -55,9 +55,23 @@ def test_ingestion_simple(memory_ds: Dataset, convert_to_pathlib: bool):
 
     assert ds["images"].meta.sample_compression == "jpeg"
     assert list(ds.tensors.keys()) == ["images", "labels"]
-    assert ds["images"].numpy().shape == (3, 200, 200, 3)
-    assert ds["labels"].numpy().shape == (3, 1)
-    assert ds["labels"].info.class_names == ("class0", "class1", "class2")
+    assert ds["images"][0:5].numpy().shape == (5, 200, 200, 3)
+    assert ds["images"].numpy().shape == (12, 200, 200, 3)
+    assert ds["labels"].numpy().shape == (12, 1)
+    assert ds["labels"].info.class_names == (
```

Comment:
why are we testing this here? There are better tests we can do in test_api without adding so many redundant images as well.